templates/repo/issue/view_title.tmpl

```diff
@@ -1,19 +1,22 @@
-<div class="sixteen wide column title">
-	<div class="issue-title" id="issue-title-wrapper">
-		{{if and (or .HasIssuesOrPullsWritePermission .IsIssuePoster) (not .Repository.IsArchived)}}
-			<button id="edit-title" class="ui basic button secondary edit-button not-in-edit">{{.locale.Tr "repo.issues.edit"}}</button>
-		{{end}}
-		<h1>
-			<span id="issue-title">{{RenderIssueTitle $.Context .Issue.Title $.RepoLink $.Repository.ComposeMetas | RenderCodeBlock}}</span>
-			<span class="index">#{{.Issue.Index}}</span>
-			<div id="edit-title-input" class="ui input gt-ml-4 gt-hidden">
+<div class="sixteen wide column title gt-mb-3">
+	<div class="issue-title gt-mb-4 gt-df gt-ac" id="issue-title-wrapper">
+		<h1 class="gt-df gt-ac gt-f1 gt-word-break">
```

Comment:
![image](https://user-images.githubusercontent.com/2114189/235465154-b3c62be0-c8dc-4e7b-b19c-9498c711116a.png)

Playing such overwriting game already caused a lot of problems.

Reply:
Yes, if I had infinite motivation, I would rewrite all these crappy styles, but I don't 😆

Also, can't unfortunatly use simple `.issue-title` because it's in use in 3 more places in other templates.

Reply:
Let me see what can be cleaned up without degradation...

Reply:
Cleaned it up a bit and used more CSS.